docs/src/content/hardhat-runner/docs/reference/solidity-support.md

```diff
@@ -13,7 +13,7 @@ These are the versions of Solidity that you can expect to fully work with Hardha
 - Any 0.5.x version starting from 0.5.1
 - Any 0.6.x version
 - Any 0.7.x version
-- Any 0.8.x version up to and including 0.8.18
+- Any 0.8.x version up to and including 0.8.20
```

Comment:
what do you think about bumping the version in other docs, like for example in:
docs/src/content/hardhat-runner/docs/guides/project-setup.md
docs/src/content/tutorial/creating-a-new-hardhat-project.md
?